qiskit_machine_learning/algorithms/classifiers/qsvc.py

```diff
@@ -0,0 +1,62 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Quantum Support Vector Classifier"""
+import logging
+
+from typing import Optional
+
+from sklearn.svm import SVC
+
+from qiskit_machine_learning.kernels.quantum_kernel import QuantumKernel
+
+logger = logging.getLogger(__name__)
+
+class QSVC(SVC):
+    r"""Quantum Support Vector Classifier.
+
+    This class shows how to use a quantum kernel for classification. The class extends
+    `sklearn.svm.SVC <https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html>`_,
+    and thus inherits its methods like ``fit`` and ``predict`` used in the example below.
+    Read more in the
+    `sklearn user guide https://scikit-learn.org/stable/modules/svm.html#svm-classification>`_.
+
+    **Example**
+
+    .. code-block::
+
+        qsvc = QSVC(quantum_kernel=qkernel)
+        qsvc.fit(sample_train,label_train)
+        qsvc.predict(sample_test)
+    """
+
+    def __init__(self, *args,
+                 quantum_kernel: Optional[QuantumKernel] = None, **kwargs):
+        """
+        Args:
+            quantum_kernel: QuantumKernel to be used for classification.
+        """
+
+        self._quantum_kernel = quantum_kernel if quantum_kernel else QuantumKernel()
+
+        super().__init__(kernel=self._quantum_kernel.evaluate, *args, **kwargs)
+
+    @property
+    def quantum_kernel(self) -> QuantumKernel:
+        """ Returns quantum kernel """
+        return self._quantum_kernel
+
+    @quantum_kernel.setter
+    def quantum_kernel(self, quantum_kernel: QuantumKernel):
+        """ Sets quantum kernel """
+        self._quantum_kernel = quantum_kernel
+        self.kernel = self._quantum_kernel
```

Comment:
```suggestion
        self.kernel = self._quantum_kernel.evaluate
```

Reply:
That works. You could also have an evaluate method in this class, that you give to SVC, where this just forwards the call to self._kernel.evaluate to do the real work. That way when self.kernel is updated it still goes to the right place because of the forwarding.